nodebuilder/node_test.go

```diff
@@ -136,6 +136,9 @@ func StartMockOtelCollectorHTTPServer(t *testing.T) (string, func()) {
 }
 
 func TestEmptyBlockExists(t *testing.T) {
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
no timeout for test?

Reply:
This test does not involve any blocking operations

Reply:
This test does not involve any blocking operations